source/interface.h

```diff
@@ -40,6 +40,12 @@ EXTERNC void webEngineViewLoad(void* webEngineView, char* url);
 EXTERNC void* webEngineViewPage(void* webEngineView);
 EXTERNC void webEnginePageSetHtml(void* webEnginePage, char* html);
 EXTERNC void webEnginePageRunJavaScript(void* webEnginePage, char* javascript, int id, fpIntChar callback);
+EXTERNC void webEnginePageSetLivecycleState(void* webEnginePage, int state);
```

Comment:
while we're at it, maybe we should just wrap all these function prototypes in
``` C++
#ifdef __cplusplus
extern "C" {
#endif
 
/* Function prototypes here.*/
 
#ifdef __cplusplus
}
#endif
```
block? it's shorter and requires no additional definitions. 

This can wait for another pull request or commit, though.